Kernel/Arch/aarch64/Interrupts.cpp

```diff
@@ -84,7 +84,7 @@ extern "C" void exception_common(Kernel::TrapFrame* trap_frame)
     auto fault_address = Aarch64::FAR_EL1::read().virtual_address;
     Processor::enable_interrupts();
 
-    constexpr bool print_state = true;
+    constexpr bool print_state = false;
```

Comment:
isn't this just dead code now? add an AK/Debug.h ifdef if you want to keep the code, or remove it.

Reply:
Right, good catch. I removed the code.